### PR TITLE
#155890684 Create Retrofit Instance

### PR DESCRIPTION
**Pull request by RutaleIvanPaul**
#### What does this PR do?
* Adds retrofit instance and consumes GitHub API to display developers in recycler view
#### Related Stories
[#155890684](https://www.pivotaltracker.com/story/show/155890684)
####Screenshot
screen shot 2018-10-24 at 10 27 47

### CODE REVIEW

File: app/src/main/java/com/example/ivanpaulrutale/convergelevelup/view/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    private RecyclerView.Adapter adapter;
+    private List<DeveloperDataMapper> listitems;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+//        listitems = new ArrayList<>();
```
Comment (archbishop12) [inline app/src/main/java/com/example/ivanpaulrutale/convergelevelup/view/MainActivity.java]: Remove commented out code and in other areas it exists.

File: app/src/main/java/com/example/ivanpaulrutale/convergelevelup/view/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+
+    @Override
+    public void DevelopersReady(List<DeveloperDataMapper> developers) {
+        adapter = new ListAdapter(developers,this);
+        setRecyclerView();
+
+        for (DeveloperDataMapper developerDataMapper: developers){
+            Log.i("RETROFIT","Developer:"+developerDataMapper.getLogin());
```
Comment (archbishop12) [inline app/src/main/java/com/example/ivanpaulrutale/convergelevelup/view/MainActivity.java:40]: Remove the logging code and in other areas it exists.

File: app/src/main/java/com/example/ivanpaulrutale/convergelevelup/model/DeveloperDataMapper.java [hunk 3]
```diff
@@ -0,0 +1,216 @@
+package com.example.ivanpaulrutale.convergelevelup.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+public class DeveloperDataMapper {
```
Comment (archbishop12) [inline app/src/main/java/com/example/ivanpaulrutale/convergelevelup/model/DeveloperDataMapper.java:5]: Do you think you need all these fields. Is it possible to pick only those you need now or maybe in the next tasks?

File: app/src/main/java/com/example/ivanpaulrutale/convergelevelup/view/MainActivity.java (hunk 1 above)
Comment (RutaleIvanPaul) [reply app/src/main/java/com/example/ivanpaulrutale/convergelevelup/view/MainActivity.java]: Removed

**Pull request merged**